README.md

```diff
@@ -1,56 +1,32 @@
+# Terraform Provider Iterative (TPI)
 
-The Iterative Provider is a Terraform plugin that enables full lifecycle
-management of computing resources for machine learning pipelines, including GPUs, from your favorite cloud vendors.
+[![docs](https://img.shields.io/badge/-docs-5c4ee5?logo=terraform)](https://registry.terraform.io/providers/iterative/iterative/latest/docs)
+[![tests](https://img.shields.io/github/workflow/status/iterative/terraform-provider-iterative/Test?label=tests&logo=GitHub)](https://github.com/iterative/terraform-provider-iterative/actions/workflows/test.yml)
+[![Apache-2.0](https://img.shields.io/badge/licence-Apache%202.0-blue)](https://github.com/iterative/terraform-provider-iterative/blob/master/LICENSE)
 
-The Iterative Provider makes it easy to:
+- **Provision Resources**: create cloud compute & storage resources without reading pages of documentation
+- **Sync & Execute**: easily sync & run local data & code in the cloud
+- **Low cost**: transparent auto-recovery from interrupted low-cost spot/preemptible instances
+- **No waste**: auto-cleanup unused resources (terminate compute instances upon job completion/failure & remove storage upon download of results)
+- **No lock-in**: switch between several cloud vendors with ease due to concise unified configuration
 
-- Rapidly move local machine learning experiments to a cloud infrastructure
-- Take advantage of training models on spot instances without losing any progress
-- Unify configuration of various cloud compute providers
-- Automatically destroy unused cloud resources (compute instances are terminated on job completion/failure, and storage is removed when results are downloaded)
+Supported cloud vendors include:
 
-The Iterative Provider can provision resources with the following cloud providers and orchestrators:
-
-- Amazon Web Services
+- Amazon Web Services (AWS)
 - Microsoft Azure
-- Google Cloud Platform
-- Kubernetes
-
-## Documentation
-
-See the [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) guide to learn how to use the Iterative Provider. More details on configuring and using the Iterative Provider are in the [documentation](https://registry.terraform.io/providers/iterative/iterative/latest/docs).
-
-## Support
-
-Have a feature request or found a bug? Let us know via [GitHub issues](https://github.com/iterative/terraform-provider-iterative/issues). Have questions? Join our [community on Discord](https://discord.gg/bzA6uY7); we'll be happy to help you get started!
+- Google Cloud Platform (GCP)
+- Kubernetes (K8s)
 
-## License
+## Usage
 
-Iterative Provider is released under the [Apache 2.0 License](https://github.com/iterative/terraform-provider-iterative/blob/master/LICENSE).
+See the [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) guide for a more detailed guide.
```

Comment:
Link to Getting Started seems like duplication.

Why the Getting Started is better than the current doc? Even if it's better - can we make the current doc to this level and get rid of the Getting Started link?

Reply:
If we plan to mirror the getting started guide in the readme, it should probably be a purposeful decision.

If not, it's probably better to link from the latter to the former or define why they should be different (e.g. different audience)

Reply:
The *Getting Started* docs are currently a very slightly more commented version of this Readme *Usage* section. In future I'd expect more links/asides/notes/FAQs to be in the *Getting Started* docs.

Always best to keep the Readme super terse (just enough for a beginner to run - no advanced edge cases/discussion), but link to the more verbose docs.

Reply:
Having the links if okay. But it feels like we are confusing users. User see a link to getting Started and some "Define task" / tutorial right after. What I should do getting started or Tutorial? What is the difference?

Reply:
moved to later (help/FAQ section)